rspec/testing_cors.md

```diff
@@ -0,0 +1,31 @@
+It's a bit tricky to test with `RSpec`, because we don't have a real server (like [Webrick](https://github.com/nahi/webrick), [Unicorn](http://unicorn.bogomips.org/) or [Puma](https://github.com/puma/puma)) between the code and the client. This usually is not a problem but it does affect in this particular case, and the [rfc3875](https://tools.ietf.org/html/rfc3875#section-4.1.18) explains why. The server translate the custom `HTTP` request headers in this way:
```

Comment:
Rather than start with "It's", it could be worth starting with "Support for CORS", including a link to a definition?